ApeWorX/ape
# feat: adding retention policy

### What I did

Adding a retention policy to remove old versions of containers from ape's ghcr



fixes: #

### How I did it

Used [snok](https://github.com/snok/container-retention-policy)

### How to verify it



This is a setup for future, have to build out previous versions and tag them to see if this will work. Currently set to dry-run

### Checklist
